src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -88,38 +91,81 @@ export class MakeCogGithub {
     logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
-    if (region) {
+    if (individual) {
+      if (region == null) region = 'individual';
       // Prepare new standalone tileset config
       layer.category = category;
       layer.minZoom = 0;
       layer.maxZoom = 32;
-      const tileSet: TileSetConfigSchema = {
+      const tileSet: ConfigTileSetRaster = {
         type: TileSetType.Raster,
         id: `ts_${layer.name}`,
+        name: layer.name,
         title: layer.title,
-        background: '#00000000',
+        background: { r: 0, g: 0, b: 0, alpha: 0 },
         category,
         layers: [layer],
       };
-      const content = await prettyPrint(JSON.stringify(tileSet, null, 2), ConfigPrettierFormat);
-      const tileSetPath = fsa.joinAll('config', 'tileset', region, `${layer.name}.json`);
-      const file = { path: tileSetPath, content };
+      const tileSetPath = fsa.joinAll('config', 'tileset', region, 'imagery', `${layer.name}.json`);
       // Github create pull request
-      await gh.createPullRequest(branch, title, botEmail, [file]);
+      this.createTileSetPullRequest(gh, branch, title, tileSetPath, tileSet);
     } else {
       // Prepare new aerial tileset config
       const tileSetPath = fsa.joinAll('config', 'tileset', `${filename}.json`);
       const tileSetContent = await gh.getContent(tileSetPath);
       if (tileSetContent == null) throw new Error(`Unable get the ${filename}.json from config repo.`);
       const tileSet = JSON.parse(tileSetContent) as ConfigTileSetRaster;
       const newTileSet = await this.prepareRasterTileSetConfig(layer, tileSet, category);
-      // skip pull request if not an urban or rural imagery
-      if (newTileSet == null) return;
-      // Github
-      const content = await prettyPrint(JSON.stringify(newTileSet, null, 2), ConfigPrettierFormat);
-      const file = { path: tileSetPath, content };
       // Github create pull request
-      await gh.createPullRequest(branch, title, botEmail, [file]);
+      this.createTileSetPullRequest(gh, branch, title, tileSetPath, newTileSet);
+    }
+  }
+
+  /**
+   * Prepare and create pull request for the elevation tileset config
+   */
+  async updateElevationTileSet(
+    filename: string,
+    layer: ConfigLayer,
+    category: Category,
+    individual: boolean,
+    region: string | undefined,
+  ): Promise<void> {
+    const gh = new GithubApi(this.repository);
+    const branch = `feat/bot-config-elevation-${this.imagery}${this.ticketBranchSuffix}`;
+    const title = `config(elevation): Add elevation ${this.imagery} to ${filename} config file.`;
+
+    // Clone the basemaps-config repo and checkout branch
+    logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
+    if (individual) {
+      if (region == null) region = 'individual';
+      // Prepare new standalone tileset config
+      layer.category = category;
```

Comment:
this modifies the source `layer` object which was passed into this function, in general we should avoid updating objects when passing them down into functions unless we are explicit that the function is designed to add to the source bojects.

`const targetLayer = {...layer, minZoom: 0, maxZoom:32 }`

Reply:
Just fixed, thank you.